### DISCUSSION TITLE
[bug] The vertical scroll bars start flickering when I put the browser on the left side

### DISCUSSION THREAD

**Discussion by zhenjiaguo** [community/community#36674] [Other Feature Feedback, Questions, & Ideas]
In this [page](https://github.com/mrdbourke/pytorch-deep-learning/blob/main/extras/exercises/00_pytorch_fundamentals_exercises.ipynb). The vertical scroll bars start flickering when I put the browser on the left or right side:

https://user-images.githubusercontent.com/29809822/196726099-754de63e-4b93-4eb8-8e3e-c0b5c0ac65c4.mp4

System info : Ubuntu 22.04
Browser: Chrome version 106.0.5249.119
Screen Resolution : 2560x1660

**Comment by ossanna16** [comment]
Hey there @ghost and thank you for reporting this bug! I'm glad you found our community 🙂
